batch7crud-roll2/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,305 @@
+body, html {
+    height: 100%;
+    position: relative;
+}
+
+.mainWrapper {
+    max-width: 100%;
+}
+
+.headerWrapper {
+    background-color: #666666;
+    padding: 30px 45px 30px 100px;
+    color: #ffffff;
+    border-bottom: 2px solid #333333;
+}
+
+.logo {
+    float: left;
+    font-size: 22px;
+}
+
+.bodyWrapper {
+    min-height: 600px;
+}
+
+.col2-left {
+    width: 100%;
+}
+
+.left {
+    width: 20%;
+    background-color: #336699;
+    min-height: 850px;
+    float: left;
+}
+
+.right {
+    float: left;
+    width: 80%;
+    min-height: 700px;
+}
+
+/*Login Page*/
+.loginBox {
+    width: 20%;
+    margin: 10% auto;
+    background-color: #336699;
+    min-height: 250px;
+    padding: 50px 5% 0 5%;
+}
+
+.loginTitle {
+    text-align: center;
+    color: #ffffff;
+    font-size: 16px;
+    font-weight: bold;
+    padding-bottom: 30px;
+}
+
+.formLabel {
+    color: #ffffff;
+    font-size: 14px;
+    clear: both;
+    float: left;
+    padding-bottom: 5px;
+}
+
+.inputText {
+    width: 100%;
+    clear: both;
+}
+
+.loginBox input[type=text], input[type=password] {
+    width: 100%;
+    float: left;
+    margin-bottom: 10px;
+    height: 30px;
+}
+
+.loginBox input[type=submit] {
+    clear: both;
+    float: right;
+    width: 50%;
+    margin-top: 5px;
+    font-family: montserrat, sans-serif;
+    height: 30px;
+}
+
+.errorMessage {
+    font-size: 12px;
+    color: #c9302c;
+}
+
+/*Left Navigation*/
+
+.left ul li {
+    border-bottom: 1px solid #336666;
+    cursor: pointer;
+}
+
+.left ul li a {
+    width: 100%;
+    font-size: 16px;
+    display: block;
+    color: #ffffff;
+    text-align: center;
+    line-height: 40px;
+}
+
+/*List Page*/
+
+.bodyTitle {
+    text-align: left;
+    width: 97%;
+    padding: 20px 0 20px 3%;
+    font-size: 18px;
+    font-weight: bold;
+    background-color: #eeeeee;
+    border-bottom: 2px solid #333333;
+}
+
+.loggedinUser {
+    font-size: 12px;
+    float: right;
+}
+
+.logout {
+    clear: both;
+    float: right;
+}
+
+.addNewEmployee {
+    padding: 20px 0 20px 3%;
+}
+
+.addNewEmployee a {
+    width: 10%;
+    background-color: #336699;
+    color: #ffffff;
+    display: block;
+    font-size: 14px;
+    text-decoration: none;
+    text-align: center;
+    line-height: 30px;
+    padding-left: 1%;
+    padding-right: 1%;
+}
+
+.employeeListTable {
+    padding-left: 3%;
+}
+
+.employeeListTable table {
+    border: 1px solid #cccccc;
+    width: 95%;
+}
+
+.employeeListTable table tr {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename selector employeeListTable to match the regular expression: ^[a-z][a-z\-]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aselector-naming-convention)